.github/workflows/build-test-deploy.yml

```diff
@@ -1053,25 +1050,13 @@ jobs:
           GCP_DEVNET_SERVICE_ACCOUNT: ${{ secrets.GCP_DEVNET_SERVICE_ACCOUNT }}
           GCP_DEVNET_SERVICE_ACCOUNT_KEY: ${{ secrets.GCP_DEVNET_SERVICE_ACCOUNT_KEY }}
 
-          # TODO: if not main -> dry run else deploy
```

Comment:
nixops scripts basically unmaintainable imho now, at least for me until something is done:
1. there is not main, there is no dryrun
2. there is not deep """typecheck""" of nix 
2.1 part started to be possible because of no mac
2.2. part was deleted with flake utils hook to allow nixops to be under system (so you could evaluate dummy package wrapping nixops and dump it). as nixops is root now (after removal of flake utils hook) not sure how to check it
3. sre did not gave me access of any kind to gce to have ability to map dynamic lazy not typed language to real data
4. i do not have experience with nixops

just FYI that that comment (and others removed earlier) were about maintain ability which is none for me now

Reply:
NixOps is not going to be maintained once the SRE team take the lead on deploying our artifacts, this is basically glue to have a devnet running before we have a proper testnet